src/status_im/contexts/onboarding/syncing/progress/view.cljs

```diff
@@ -70,12 +71,13 @@
 
 (defn view
   [in-onboarding?]
-  (let [pairing-status    (rf/sub [:pairing/pairing-status])
-        logged-in?        (rf/sub [:multiaccount/logged-in?])
-        pairing-progress? (pairing-progress pairing-status)
-        profile-color     (or (:color (rf/sub [:onboarding/profile]))
-                              (rf/sub [:profile/customization-color]))]
-    [rn/view {:style (style/page-container in-onboarding?)}
+  (let [{:keys [top bottom]} (safe-area/get-insets)
+        pairing-status       (rf/sub [:pairing/pairing-status])
```

Comment:
Do you think we should create a reframe subscription for this value if we're not using a hook?

Reply:
I don't think so, it's just a value gathered when the app starts, we don't need to glorify it :thinking: